components/Tooltip/Tooltip.jsx

```diff
@@ -43,17 +43,18 @@ const Tip = styled.div`
 
 const TipText = styled.span`
   display: inline-block;
-  max-width: 250px;
```

Comment:
This line has been removed for the Tooltip component not to truncate text.

Reply:
what do you think to switch `max-width` to `min-width`? If you use it, you don't need remove this and you don't need to set `max-content` in line https://github.com/catho/quantum/pull/233/files#r431827523

Reply:
min-width change the expected layout 
![image](https://user-images.githubusercontent.com/36733015/83406530-36c5ea80-a3e5-11ea-9227-79c711f603bb.png)

Reply:
This change is breaking the layout:
![image](https://user-images.githubusercontent.com/1851195/83418562-2ec47580-a3fa-11ea-839b-9cda0dc37a60.png)

Before the change:
![image](https://user-images.githubusercontent.com/1851195/83418766-806d0000-a3fa-11ea-9392-0157535b68f5.png)

Reply:
Why don't we add a prop to change the behavior for the case that you are addressing and keep the original behavior when not using this prop?